app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,62 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        setItemMessage(item, response);
+      }
+    })
+  });
+
+  $(".proxy-status, .compute-status").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'post',
+      url:  url,
+      success: function(response) {
+        setItemStatus(item, response);
+      }
+    })
+  });
+
+  $(".proxy-show-status").each(function(index, item) {
+      var item = $(item);
+      var url = item.data('url');
+      $.ajax({
+        type: 'post',
+        url:  url,
+        success: function(response) {
+          setItemStatus(item, response);
+        }
+      })
+    });
+});
+
+function setItemStatus(item, response) {
+  if(response.status == "OK") {
+    item.attr('title', __('Active'));
+    item.addClass('text-success');
+    item.html('<span class="glyphicon glyphicon-ok-sign" />');
+  } else {
+    item.attr('title', response.message);
+    item.addClass('text-danger')
+    item.html('<span class="glyphicon glyphicon-exclamation-sign" />');
+  }
+  item.tooltip({html: true});
+}
+
+function setItemMessage(item, response) {
+  item.attr('title',response.message);
+  if(response.success == true) {
+    item.addClass('text-success')
+    item.text(__(response.message));
```

Comment:
It's probably preferable to just translate it in the server's response if you can, since the locale is already set.  Occasionally there's a reason to send an English string to translate on the client, but there's no point here AFAICT.